data/last-news.yml

```diff
@@ -1,8 +1,8 @@
-- title: "Kotlin Features Survey Edition #2"
-  link: https://blog.jetbrains.com/kotlin/2021/06/kotlin-features-survey-edition-2/
+- title: "Improved Kotlin Experience in IntelliJ IDEA 2021.2"
+  link: https://blog.jetbrains.com/kotlin/2021/07/improved-kotlin-experience-intellij-idea-2021-2/
 
-- title: "Kotlin 1.5.20 Released"
-  link: https://blog.jetbrains.com/kotlin/2021/06/kotlin-1-5-20-released/
+- title: "First Preview of Kotlin 1.5.30"
+  link: https://blog.jetbrains.com/kotlin/2021/07/kotlin-1-5-30-rc-released/
```

Comment:
That link is broken. I guess that there is a mistake and Katia implies that the link should be https://blog.jetbrains.com/kotlin/2021/07/kotlin-1-5-30-m1-released/.

Reply:
Link is correct, but release isn't ready yet. Also the date was wrong, fixed